### DISCUSSION TITLE
pypi安装的paddleocr2.7版本ser_re缺少相关文件,SerRePredictor不能初始化

### DISCUSSION THREAD

**Discussion by ruisv** [PaddlePaddle/PaddleOCR#12494] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：fedora x64
- 版本号/Version：Paddle：2.4.2  PaddleOCR：2.7 问题相关组件/Related components：SerRePredictor
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):

请尽量不要包含图片在问题中/Please try to not include the image in the issue.
`from tools.infer_kie_token_ser_re import make_input`
pypi安装的包并没有tools.infer_kie_token_ser_re这个文件

**Comment by SWHL** [comment]
如果问题仍然存在，请安装最新paddleocr版本，再次尝试
